rulings/cards-rulings.yaml

```diff
@@ -2722,6 +2725,7 @@
   - Must be played after the presses are handled and can be played if the round ends prematurely. [RTR 20001020] [LSJ 20001024] [RTR 20030519]
   - Can be played before or after effects that are played at the end of the round or "when the combat would end" (eg. [AUS] {Telepathic Tracking}). Can be played after [CEL] {Psyche!} [LSJ 20021113] [ANK 20191219] [ANK 20180910-1]
   - Can be played if the opponent was burned. [LSJ 20031201-2]
+  - You can play it if the opposing vampire has not lost any blood to "cycle". [BCH 20240228]
```

Comment:
![image](https://github.com/lionel-panhaleux/krcg/assets/610753/b23e94b1-9e2b-461d-a731-33a230403fd2)

Reply:
Ah, I see. It's PDF-only T.T

Reply:
Yep I noticed this. Maybe we should wait with merging and ask Vincent to update the github.
Then we refer to github as the permalink (since we can use the revision and line).